Resolve syncing conflicts from master-sync-work to master

openpublishbuild opened the pull request:
Pull request opened by OPS to resolve syncing conflicts from master-sync-work to master.

Commit:
Consistently use braces for single line blocks

In some cases you where using {} for single line statements following conditionals such as if.  In other cases they where omitted.  As per StyleCop guidelines the usage should be consistent, and using {} is recommended.

Commit:
Removed RC from Visual Studio 2017 product name

Commit:
Merge pull request #1421 from rpetrusha/csproj

Removed RC from Visual Studio 2017 product name

Commit:
Redirects for Visual Studio 2015 tutorials (#1423)

* Redirects for Visual Studio 2015 tutorials

* fixed broken link

* Changed 'updated' to 'retired'

mairaw committed:
Update .openpublishing.publish.config.json

saldana committed:
Merge pull request #1431 from dotnet/pdf

Update .openpublishing.publish.config.json

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C21%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701210041393371-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
fixed broken link (#1432)

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C21%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701210111171796-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
added links between versions (#1424)

* added links between versions

* updating links to nuget and asp.net

* addressed feedback

* fixed broken link

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C21%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701210755452880-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

bleroy committed:
Small tweaks to the full VS 2017 Core tutorial

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C24%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701240048423230-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
updated title (#1443)

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C24%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701240107595884-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

BillWagner committed:
Reorganize the C# Guide TOC (#1318)

* Use a sub-TOC for the Tour

This is a first experiment with the sub TOC. If I like the way this
works, I’ll use it throughout the C# Guide

* Update GettingStarted, Tutorials

* Update C# Interactive TOC

* move language reference.

* keywords and operators TOC updates.

* update preprocessor section

* update compiler options

* fix OPS build errors

* finish the language reference.

* top level C# programming guide

This needs to be broken down much more in the lower levels.

* update inside a C# program.

* use sub-toc for arrays.

* use sub tor for classes and structs

* update the delegates section

* update events

* update exceptions TOC

* post-merge updates

* update the file system section.

* update the generics section

* update from build errors

* update the interop section

* update file names.

* update section on main

* update XML doc section.

* revert incorrect file name changes.

* update strings and types.

* update concepts

* fix a global replace error

* update assemblies and gas

* update concepts async section

* fix build errors from global search / replace

* co-contra variance, expression trees

* update the linq topics.

* all sub TOCs are in place.

* update sub-toc after rebase.

* fix a rename from a merge conflict.

* fix a build error.

* one final build warning...

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C24%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701240420566532-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
Merge branch 'csproj' into master

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C24%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701240619238122-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Add braces to single-line statement in line 16

mairaw committed:
reverting merge (#1447)

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C24%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701241505362659-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Getting Started with F# in Visual Studio Code with Ionide: add visual aid for an often missed step (#1457)

* "Create new directory" visual aid

* Added "open folder" visual aid to article

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C25%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701251643157843-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

IEvangelist committed:
Minor correction in the opening sentence on Tuples (#1458)

The first sentence needed to be corrected.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C26%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701261402129993-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
fixed typos (#1459)

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C26%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701261813098962-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
preparing changes for RC3 release

a-ctor committed:
Added a missing bracket in a code example

Commit:
Merge pull request #1404 from quintusm/patch-2

Consistently use braces for single line blocks

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C30%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701301644034894-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Merge pull request #1465 from a-ctor/patch-1

Added a missing bracket in a code example

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C30%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701302125021942-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Revision of interpolated strings topic.

TingluoHuang committed:
fix dotnet test --nobuild option, should be --no-build (#1466)

* fix --nobuild option, should be --no-build

* Update dotnet-test.md

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C31%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701310312263714-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
Merge pull request #1473 from mairaw/rc3

changing tooling version name to RC3

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C31%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701310746235662-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
readding deleted files (#1461)

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C31%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701311345443861-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Corrected publishing errors.

mairaw committed:
removing caps link (#1463)

* removing caps link

* addressed feedback

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C31%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701311845542684-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Addressed inconsistent terminology, fixed formatting.

mairaw committed:
fix broken links (#1490)

* fix broken links

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C2%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702022341478427-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

cmbrown1598 committed:
Update snippet6902.fs (#1496)

Corrected usage of TryParse.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702030101348575-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

2moveit committed:
Added required NuGet package Microsoft.WindowsAzure.ConfigurationManager

Required for CloudConfigurationManager namespace Microsoft.Azure.

afzaal-ahmad-zeeshan committed:
Update whats-new.md

Fixed a small typo.

BillWagner committed:
path 1 to update the TOC for LINQ concepts (#1492)

* path 1 to update the TOC for LINQ concepts

* fix warnings.

* missed one warning.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702031355167650-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mcroley committed:
Generic Types update

Generic Type description indicates an example but an example was not included

Commit:
Merge pull request #1500 from afzaal-ahmad-zeeshan/patch-1

Updated whats-new.md

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702031624447197-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mairaw committed:
Merge pull request #1499 from 2moveit/2moveit-blobs

Added required NuGet package Microsoft.WindowsAzure.ConfigurationManager

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702031803326857-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Corrected left-right confusion.

Commit:
Changed date in metadata.

mcroley committed:
Update index.md

Commit:
Corrected bad article.

Commit:
Merge pull request #1469 from rpetrusha/interpolated

Revision of interpolated strings topic.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702032235124707-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

tdykstra committed:
Add notes about inline code styling (#1502)

* Add notese about inline code styling

* respond to feedback

Commit:
Merge pull request #1501 from mcroley/patch-1

Code Example Included with Generic Types

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702032245127681-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702032246064852-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Merge pull request #1504 from rpetrusha/leftright

Corrected left-right confusion.

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C3%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702032311413191-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

herohua committed:
resolve syncing conflicts

openpublishtest commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/using-exceptions.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/using-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/using-exceptions?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-file-or-folder?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-read-from-a-text-file.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-read-from-a-text-file.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-read-from-a-text-file?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-write-to-a-text-file.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-write-to-a-text-file.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-write-to-a-text-file?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/inside-a-program/hello-world-your-first-program?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/example-com-class.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/example-com-class.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/example-com-class?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/interoperability.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/interoperability.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/interoperability?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/walkthrough-office-programming.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/walkthrough-office-programming.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/walkthrough-office-programming?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/command-line-arguments?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/how-to-display-command-line-arguments.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/how-to-display-command-line-arguments.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/how-to-display-command-line-arguments?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/main-and-command-line-arguments.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/main-and-command-line-arguments.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/main-and-command-line-arguments?branch=pr-en-us-1) |
[docs/csharp/programming-guide/main-and-command-args/main-return-values.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/main-and-command-args/main-return-values.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/main-and-command-args/main-return-values?branch=pr-en-us-1) |
[docs/csharp/misc/cs1558.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs1558.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs1558?branch=pr-en-us-1) |
[docs/csharp/misc/cs1559.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs1559.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs1559?branch=pr-en-us-1) |
[docs/csharp/misc/cs5001.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs5001.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs5001?branch=pr-en-us-1) |
[docs/csharp/programming-guide/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/classes-and-structs/methods.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/classes-and-structs/methods.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/classes-and-structs/methods?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/async-return-types.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/async-return-types.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/async-return-types?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/control-flow-in-async-programs.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/control-flow-in-async-programs.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/control-flow-in-async-programs?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/fine-tuning-your-async-application.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/fine-tuning-your-async-application.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/fine-tuning-your-async-application?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/using-async-for-file-access.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/using-async-for-file-access.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/using-async-for-file-access?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/modifiers/assembly.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/modifiers/assembly.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/modifiers/assembly?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/modifiers/module-keyword.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/modifiers/module-keyword.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/modifiers/module-keyword?branch=pr-en-us-1) |
[docs/csharp/programming-guide/statements-expressions-operators/lambda-expressions.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/statements-expressions-operators/lambda-expressions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/statements-expressions-operators/lambda-expressions?branch=pr-en-us-1) |
[docs/csharp/programming-guide/statements-expressions-operators/statements.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/statements-expressions-operators/statements.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/statements-expressions-operators/statements?branch=pr-en-us-1) |
[docs/csharp/programming-guide/types/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/types/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/types/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers?branch=pr-en-us-1) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C6%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702060234431354-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

openpublishtest commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/csharp/misc/cs0255.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs0255.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs0255?branch=pr-en-us-1) |
[docs/fsharp/using-fsharp-on-azure/blob-storage.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/fsharp/using-fsharp-on-azure/blob-storage.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/fsharp/using-fsharp-on-azure/blob-storage?branch=pr-en-us-1) |
[docs/csharp/programming-guide/classes-and-structs/methods.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/classes-and-structs/methods.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/classes-and-structs/methods?branch=pr-en-us-1) |
[docs/csharp/misc/cs0155.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs0155.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs0155?branch=pr-en-us-1) |
[docs/csharp/misc/cs0156.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs0156.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs0156?branch=pr-en-us-1) |
[docs/csharp/misc/cs0157.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs0157.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs0157?branch=pr-en-us-1) |
[docs/csharp/misc/cs0160.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/misc/cs0160.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs0160?branch=pr-en-us-1) |
[docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/inside-a-program/hello-world-your-first-program?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/example-com-class.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/example-com-class.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/example-com-class?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/interoperability.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/interoperability.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/interoperability?branch=pr-en-us-1) |
[docs/csharp/programming-guide/interop/walkthrough-office-programming.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/interop/walkthrough-office-programming.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/interop/walkthrough-office-programming?branch=pr-en-us-1) |
[docs/samples-and-tutorials/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/samples-and-tutorials/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/samples-and-tutorials/index?branch=pr-en-us-1) |
[docs/standard/managed-code.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/standard/managed-code.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/standard/managed-code?branch=pr-en-us-1) |
[docs/standard/native-interop.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/standard/native-interop.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/standard/native-interop?branch=pr-en-us-1) |
[docs/standard/base-types/composite-format.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/standard/base-types/composite-format.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/standard/base-types/composite-format?branch=pr-en-us-1) |
[docs/visual-basic/programming-guide/language-features/statements.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/programming-guide/language-features/statements.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/language-features/statements?branch=pr-en-us-1) |
[docs/fsharp/tutorials/getting-started/getting-started-vscode.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/fsharp/tutorials/getting-started/getting-started-vscode.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-1) |
[docs/csharp/language-reference/compiler-options/target-winmdobj-compiler-option.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/language-reference/compiler-options/target-winmdobj-compiler-option.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/language-reference/compiler-options/target-winmdobj-compiler-option?branch=pr-en-us-1) |
[docs/csharp/language-reference/keywords/try-catch.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/language-reference/keywords/try-catch.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/language-reference/keywords/try-catch?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/error-messages/extension-attribute-can-be-applied-only-to-module-sub-or-function-declarations.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/error-messages/extension-attribute-can-be-applied-only-to-module-sub-or-function-declarations.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/extension-attribute-can-be-applied-only-to-module-sub-or-function-declarations?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/error-messages/first-statement-of-sub-new-must-be-explicit-call-to-mybase-new-or-myclass-new.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/error-messages/first-statement-of-sub-new-must-be-explicit-call-to-mybase-new-or-myclass-new.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/first-statement-of-sub-new-must-be-explicit-call-to-mybase-new-or-myclass-new?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/error-messages/identifier-expected.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/error-messages/identifier-expected.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/identifier-expected?branch=pr-en-us-1) |
[docs/csharp/programming-guide/xmldoc/exception.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/xmldoc/exception.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/xmldoc/exception?branch=pr-en-us-1) |
[docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/language-features/procedures/extension-methods?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/start-multiple-async-tasks-and-process-them-as-they-complete?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/using-async-for-file-access.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/using-async-for-file-access.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/using-async-for-file-access?branch=pr-en-us-1) |
[docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/statements/parameter-list.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/statements/parameter-list.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/statements/parameter-list?branch=pr-en-us-1) |
[docs/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/compiler-generated-exceptions.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/compiler-generated-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/compiler-generated-exceptions?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/creating-and-throwing-exceptions.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/creating-and-throwing-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/creating-and-throwing-exceptions?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/exception-handling.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/exception-handling.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/exception-handling?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/exceptions-and-exception-handling.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/exceptions-and-exception-handling.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/exceptions-and-exception-handling?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/index.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/index?branch=pr-en-us-1) |
[docs/csharp/programming-guide/exceptions/using-exceptions.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/exceptions/using-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/using-exceptions?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-file-or-folder?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-get-information-about-files-folders-and-drives?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-iterate-through-a-directory-tree?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-provide-a-progress-dialog-box-for-file-operations?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-read-a-text-file-one-line-at-a-time?branch=pr-en-us-1) |
[docs/csharp/programming-guide/file-system/how-to-read-from-a-text-file.md](https://github.com/openpublishtest/docs/blob/master-sync-work/docs/csharp/programming-guide/file-system/how-to-read-from-a-text-file.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-read-from-a-text-file?branch=pr-en-us-1) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C6%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702060239141163-1%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
